test.js

```diff
@@ -14,48 +14,56 @@ test('.get() - Sublime', t => {
 	t.is(m.get('subl').id, 'sublime');
 	t.is(m.get('Sublime Text').id, 'sublime');
 	t.is(m.get('/Applications/Sublime Text.app/Contents/SharedSupport/bin/subl').id, 'sublime');
+	t.is(m.get('sublime').isTerminalEditor, false);
```

Comment:
There's no point in testing this property. It's always going to be what we defined. The other checks are there to make sure we match the correct editor.